Xavi dev

Xavierito1825 opened the pull request:
Creacion de la estructura de Pagina de reservas y reserva (individual).
Pagina reserva (individual) --> Editor de reserva

Xavierito1825 committed:
estructura de reservas, comienzo de reserva (Individual)

Xavierito1825 committed:
acabado editor de reservas

alfman99 reviewed (changes requested):
No entiendo porque has puesto una pagina de reserva y no tiene nada que ver con una reserva? aqui estoy viendo cosas de un perfil

alfman99 commented on interfaces.d.ts:
```diff
@@ -63,4 +63,11 @@ export interface PerfilData {
+}
+
+export interface ReservaData {
+  hora: string;
```
esta hora deberia ser un tipo Date

alfman99 reviewed (commented):
vale, esto mejor

alfman99 committed:
Este archivo lo ha hecho el Marc

https://github.com/PTIN2022/GeSyS-Front/blob/1a80b44e4747261fb888618d32eb537046dcdd47/pages/admin/reservas/index.tsx

Xavierito1825 committed:
solucionado el tema Fecha Date, Incuida Duracion

Xavierito1825 committed:
Merge branch 'xavi_dev' of https://github.com/PTIN2022/GeSyS-Front into xavi_dev
